plugins/vault-backend/config.d.ts

```diff
@@ -44,5 +45,11 @@ export interface Config {
      * The version of the K/V API. Defaults to `2`.
      */
     kvVersion?: 1 | 2;
+
+    /**
+     * If defined, it sets the schedule used to refresh the vault token
+     * @visibility backend
```

Comment:
```suggestion
```

It's the default, so redundant. It's also potentially confusing as it will only apply to the `schedule` object itself, no fields within it.

Reply:
Removed